### PR TITLE
Add toggle for sample values

### PR DESCRIPTION
**Pull request by sfc-gh-jhilgart**
Add in a flag to dictate the number of sample values per column. Default to 10 with a floor of 1.

### CODE REVIEW

File: semantic_model_generator/snowflake_utils/snowflake_connector.py [hunk 1]
```diff
@@ -217,11 +218,9 @@ class SnowflakeConnector:
     def __init__(
         self,
         account_name: str,
-        ndv_per_column: int = 0,
```
Comment (sfc-gh-jhilgart) [inline semantic_model_generator/snowflake_utils/snowflake_connector.py:220]: unused so remove

File: semantic_model_generator/generate_model.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 
 _PLACEHOLDER_COMMENT = "  "
 _FILL_OUT_TOKEN = " # <FILL-OUT>"
+_DEFAULT_N_SAMPLE_VALUES_PER_COL = 10
```
Comment (sfc-gh-rehuang) [inline semantic_model_generator/generate_model.py:24]: can we make it to 3?
Or to your earlier point have it be 3 for non-string type and only 10 for string type?

**Comment by sfc-gh-rehuang** [comment]
can you also follow up to allow input # of sample values into streamlit app? Thanks!

File: semantic_model_generator/generate_model.py (hunk 2 above)
Comment (sfc-gh-jhilgart) [reply semantic_model_generator/generate_model.py:24]: Let's do that as a fast follow. 

I think we should leave as 10 and let clients update? Especially with retrieval launching, more should be better.

**Review by sfc-gh-rehuang — APPROVED** [review]

**Pull request merged**